packages/babel-plugin-fbtee/src/bin/FbtCollector.tsx

```diff
@@ -29,9 +29,8 @@ export type CollectorConfig = {
   transform?: ExternalTransform | null;
 };
 type ParentPhraseIndex = number;
-export type ChildParentMappings = {
-  [childPhraseIndex: number]: ParentPhraseIndex;
-};
+export type ChildParentMappings = Map<number, ParentPhraseIndex>;
```

Comment:
`Map`s serialize to an empty object, e.g. compare the `.source_strings.json` in the example app using this branch vs main

Main
```
 "childParentMappings": {
  "14": 13,
  "23": 22,
  "24": 23
 },
 ```
 
 This branch
 ```
  "childParentMappings": {},
 ```
 
 I assume this is unintended, but not really sure what `childParentMappings` is used for in the source strings payload

Reply:
🤦‍♂️ You are right – and there are no tests for this behavior. I'll fix it and add a test.

Reply:
Thank you for the thorough explanation!